includes/templates/bootstrap/templates/tpl_index_product_list.php

```diff
@@ -15,7 +15,10 @@
  */
 ?>
 <div id="indexProductList" class="centerColumn">
+
+<?php if (!empty($current_categories_name)) { ?>
     <h1 id="indexProductList-pageHeading" class="pageHeading"><?php echo $current_categories_name; ?></h1>
+<?php } ?>
```

Comment:
I don't agree with this one; if a category's name is an empty string, the store's got more problems than accessibility.

Reply:
I might have gotten carried away.  No objections to dropping this - looking at a live site where I was testing, I did not make this change.